apis/go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/kubernetes-csi/external-snapshotter/apis/v2
```

Comment:
Can you move "apis" folder under pkg/client and keep them in the same package?

Reply:
@xing-yang sure i will update this